### DISCUSSION TITLE
Define multiple path for video file

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5454] [Q&A] [answered]
I  store my entire video collection on external harddrive that is not connected to my laptop usually. If I want to watch a bunch of videos in next couple of days, I copy (not move `mv`) those  videos from my harddrive to laptop. So, I create duplicate of those videos on my laptop from Harddrive.

I defined both  those location in Library (in the following `/mnt/Secure/Video` is on my Laptop and other path on external harddrive).

![image](https://github.com/user-attachments/assets/fc689304-8a90-41e3-873c-5f3e27b53c35)

After copying I did the scan as follows, but the path of the video is not getting updated and the video doesn't plays

![image](https://github.com/user-attachments/assets/3b4c6629-06d5-4029-85eb-c791613f0dcd)

We can see in the following image that it shows only the path on the external harddrive.

![image](https://github.com/user-attachments/assets/7efaefe6-514e-448c-9118-97b55408f87b)

How to make sure that copies of video would update and store both paths, so that I can play it when I copy it on my laptop?

**Comment by ariv-x** [comment]
Look closely -- it says "File Info (2)". That means it has found both paths. It just isn't smart about it. Look a little further down and click to expand the other file (with the local path), and make that one primary.

Unfortunately I do not know of any way to do this in bulk. But you can do it as you get to each file.

**Reply by lamyergeier** [reply]
@ariv-x  As can be seen below only one path is visible. Could you please tell what I need to expand? (when I click **File Info (2)** or **Path**, nothing happens) 

![image](https://github.com/user-attachments/assets/70d43071-5ac5-4acc-bda7-5dbd6f0236f9)

**Reply by ariv-x** [reply, marked as answer]
It _should_ show right below that. It will be just the filename in a little box, just like "BelAmi - Hoy Kogan and Matt Thurman.mp4". Click on that and it will expand.

**Reply by lamyergeier** [reply]
@ariv-x Thanks! I wish this could be automated. Surprisingly, I have already excluded the `WD1` path in Scene search window, and still that path shows up as the default path.
